### PR TITLE
Set default of confidence scores to none

### PR DESCRIPTION
**Pull request by neubig**
Fixes https://github.com/neulab/ExplainaBoard/issues/135

Also, this renames `confidence_score_up` to `confidence_score_high` so it's congruent with `confidence_score_low`.

### CODE REVIEW

File: explainaboard/metric.py [hunk 1]
```diff
@@ -51,19 +50,18 @@ def mean_confidence_interval(data, confidence=0.95):
...
 
         if not self._is_print_confidence_interval:
-            self._results["confidence_score_low"] = 0
-            self._results["confidence_score_up"] = 0
+            self._results["confidence_score_low"] = None
+            self._results["confidence_score_high"] = None
         else:
             (
                 confidence_interval_low,
                 confidence_interval_up,
```
Comment (OscarWang114) [inline explainaboard/metric.py:61]: Let's rename `confidence_interval_up` as well.

File: explainaboard/builders/conditional_generation.py [hunk 2]
```diff
@@ -239,13 +239,13 @@ def get_overall_performance(
         for metric_name in sys_info.metric_names:
 
             overall_value = self.score_dic["corpus_level"]["corpus_" + metric_name]
-            confidence_score_low = 0.0
-            confidence_score_up = 0.0
+            confidence_score_low = None
```
Comment (OscarWang114) [inline explainaboard/builders/conditional_generation.py:242]: Perhaps we can remove this variable and do (same for high):
```
Performance = (
...
confidence_score_low=None
...
)
```
This is more consistent with your code in other places. 😃

**Review by OscarWang114 — CHANGES_REQUESTED** [review]
@neubig Thanks for the PR! I requested two small changes.

**Comment by neubig** [comment]
Thanks! I'll fix the second one in #138 because it'll cause merge conflicts if I fix it here.

**Pull request merged**